setup.sh

```diff
@@ -437,6 +446,13 @@ if [ -f weather.sh ]; then
     ./weather.sh setup
 fi
 
+# Set up Sun and Moon data provider
+if [ -f grafana/sunandmoon.yml ]; then
+    cp grafana/sunandmoon.yml grafana/provisions/datasources/sunandmoon.yml
+    sed -i "s@zzLAT@${LAT}@g" grafana/provisions/datasources/sunandmoon.yml
+    sed -i "s@zzLONG@${LONG}@g" grafana/provisions/datasources/sunandmoon.yml
+fi
+
```

Comment:
Intent here is to keep grafana/sunandmoon.yml as a template (git controlled) and create the localized version in the datasources directory that has the correct LAT and LONG.

Reply:
Makes sense. Is it worth naming it `sunandmoon-template.yml` to avoid confusion?

Reply:
Great idea!